DESCRIPTION

```diff
@@ -23,6 +23,7 @@ LinkingTo:
 Suggests:
     knitr,
     rmarkdown,
+    pkgload,
```

Comment:
```suggestion
```

I don't think we need this here?

Reply:
Yes, I think that was only there for when I first added the demo scripts. Should be fine to remove.